tensorflow_probability/python/positive_semidefinite_kernels/polynomial_test.py

```diff
@@ -82,17 +95,18 @@ def testNoneShapes(self):
     self.assertEqual([2, 3, 2], k.batch_shape.as_list())
 
   @parameterized.named_parameters(
-      ('Shape [] kernel', 2., 2., 2., []),
-      ('Shape [1] kernel', [2.], [2.], [2.], [1]),
-      ('Shape [2] kernel', [1., 2.], [1., 2.], [1., 2.], [2]),
+      ('Shape [] kernel', 2., 2., 2., 2., []),
```

Comment:
these expressions are a bit hard to parse mentally. i think these parameter entries can be kwargs -- could you add them (matching the method kwargs), which i think will also force one arg per line and be a bit more readable?